BeeKit/Model/DataPoint.swift

```diff
@@ -78,6 +78,7 @@ public class DataPoint: NSManagedObject {
         value = json["value"].numberValue
         comment = json["comment"].stringValue
         requestid = json["requestid"].stringValue
+        updatedAt = json["updated_at"].intValue
```

Comment:
This is meant to be called to update a datapoint's data. Shall we check that the ids match? That is expected, correct?